src/game.cpp

```diff
@@ -316,6 +316,85 @@ static inline u8 getNeighbors(v3s16 p, INodeDefManager *nodedef, ClientMap *map,
 	return neighbors;
 }
 
+namespace {
+class Interpolator

```

Comment:
Not a single line of documentation about this class and its methods? Thats bad, especially as the name is so generic.